deps/rabbit/src/rabbit_message_interceptor.erl

```diff
@@ -1,50 +1,49 @@
-%% This Source Code Form is subject to the terms of the Mozilla Public
-%% License, v. 2.0. If a copy of the MPL was not distributed with this
-%% file, You can obtain one at https://mozilla.org/MPL/2.0/.
-%%
-%% Copyright (c) 2007-2025 Broadcom. All Rights Reserved. The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries. All rights reserved.
-
-%% This module exists since 3.12 replacing plugins rabbitmq-message-timestamp
-%% and rabbitmq-routing-node-stamp. Instead of using these plugins, RabbitMQ core can
-%% now be configured to add such headers. This enables non-AMQP 0.9.1 protocols (that
-%% do not use rabbit_channel) to also add AMQP 0.9.1 headers to incoming messages.
 -module(rabbit_message_interceptor).
--include("mc.hrl").
 
--export([intercept/1]).
+-export([intercept/3,
+         set_msg_annotation/4]).
+
+-type protocol() :: amqp091 | amqp | mqtt310 | mqtt311 | mqtt50.
```

Comment:
change `amqp` to `amqp10` as it then matches the global metric labels we currently emit